tests/aequilibrae/distribution/test_gravityCalibration.py

```diff
@@ -30,38 +35,42 @@
 
 
 class TestGravityCalibration(TestCase):
-    def setUp(self):
-        # GravityCalibration requires an object that has a `parameters` attribute. `Parameters` fits
-        # this requirement, so that we don't need to create a full project
-        self.proj = Parameters()
-
     def test_calibrate(self):
+        par = Parameters()
+
         args = {"impedance": impedance, "matrix": matrix, "function": "power", "nan_to_zero": False}
 
-        distributed_matrix = GravityCalibration(self.proj, **args)
+        distributed_matrix = GravityCalibration(par, **args)
         distributed_matrix.calibrate()
         if distributed_matrix.gap > 0.0001:
             self.fail("Calibration did not converge")
 
     def test_calibrate_with_omx(self):
-        imped = AequilibraeMatrix()
-        imped.load(siouxfalls_skims)
+        self.proj_dir = os.path.join(gettempdir(), uuid.uuid4().hex)
```

Comment:
@jamiecook This is what fixes it. Instead of opening the matrix from the data folder, we copy to temp and open from there